scripts/deployment/goerli-testnets.json

```diff
@@ -0,0 +1,61 @@
+{
+  "L1": {
+    "name": "Goerli",
+    "rpc": "https://goerli.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161",
+    "chain_id": 5
+  },
+  "L2": [
+    {
+      "name": "Arbitrum Goerli",
+      "rpc": "https://goerli-rollup.arbitrum.io/rpc",
+      "chain_id": 421613,
+      "l1_messenger": [
+        "ArbitrumL1Messenger",
+        "0xaf4159A80B6Cc41ED517DB1c453d1Ef5C2e4dB72",
+        "0x6BEbC4925716945D46F0Ec336D5C2564F419682C"
+      ],
+      "l2_messenger": "ArbitrumL2Messenger",
+      "finality_period": 604800,
+      "request_manager_arguments": {
+        "claim_stake": 0.00047,
+        "claim_request_extension": 3600,
+        "claim_period": 3600,
+        "challenge_period_extension": 3600
+      }
+    },
+    {
+      "name": "Optimism Goerli",
+      "rpc": "https://goerli.optimism.io",
+      "chain_id": 420,
+      "l1_messenger": [
+        "OptimismL1Messenger",
+        "0x5086d1eEF304eb5284A0f6720f79403b4e9bE294"
+      ],
+      "l2_messenger": "OptimismL2Messenger",
+      "finality_period": 604800,
```

Comment:
Just to make sure: is this finality period correct?

Reply:
For Optimism on Goerli:
- L1 -> L2 = 5 minutes (https://community.optimism.io/docs/developers/bridge/messaging/#for-ethereum-l1-to-optimism-l2-transactions)
- L2 -> L1 = 5 minutes (https://community.optimism.io/docs/developers/bridge/messaging/#for-optimism-l2-to-ethereum-l1-transactions)

Reply:
Boba should be the same

Reply:
✅